apps/newsletters-api/env.local.example.txt

```diff
@@ -39,3 +39,10 @@ USER_PERMISSIONS={"software.developer@guardian.co.uk": 0}
 # whether to fetch from a local instance of email rendering on port 3010
 # instead of the PROD service.
 # USE_LOCAL_EMAIL_RENDERING=true
+
+# toggle whether to send email notifications
+ENABLE_EMAIL_SERVICE=false
+
+# The email addresses to which all emails are sent to when not on PROD.
+# use semi-colons to separate the values.
+# TEST_EMAIL_RECIPIENTS=firstname.lastname@guardian.co.uk;other.person@guardian.co.uk;
```

Comment:
Do we think this allows sufficient control over who receives emails?

I have been away so perhaps this has already been discussed? In this model, everyone gets every message - I can see that _we_ might want every message, but CP might not? 

We might want to think about whether we define email groups in config that will allow us to include / exclude recipients programatically - That could come later, though - not necessarily for this PR

Reply:
The TEST_EMAIL_RECIPIENTS are only used when not on PROD (IE in DEV or CODE). In PROD, the recipients are determined by what type of messages it is - the function for building the message passes the list of `prodRecipients` to `getMessageConfig`.
At them moment, all the messages would send to 'newsletters.dev@guardian.co.uk' on PROD if the service was enabled, but we'll be able set appropriate recipients for different messages.

Reply:
Ahh gotcha - misread the intent there 👍